x-pack/plugins/ml/public/ui_actions/index.ts

```diff
@@ -68,8 +72,8 @@ export function registerMlUiActions(
   uiActions.registerAction(categorizationADJobAction);
 
   // Assign triggers
+  uiActions.addTriggerAction('ADD_PANEL_TRIGGER', addSingleMetricViewerPanelAction);
   uiActions.addTriggerAction('ADD_PANEL_TRIGGER', addSwimlanePanelAction);
-
   uiActions.addTriggerAction(CONTEXT_MENU_TRIGGER, editSwimlanePanelAction);
   uiActions.addTriggerAction(CONTEXT_MENU_TRIGGER, editSingleMetricViewerPanelAction);
```

Comment:
Testing https://github.com/elastic/kibana/pull/180544/commits/3081445c7ca8b695b64cd157e42258698662d708 I don't see the 'Edit single metric viewer' action. Is this still part a WIP?

Reply:
Good catch - edit action compatibility check has been fixed in b8beb881925a614f11c0a8f05e8055ef24fc56dd